ec/ecdatasource/deploymentdatasource/schema_apm.go

```diff
@@ -71,7 +71,7 @@ func apmTopologySchema() *schema.Schema {
 					Type:     schema.TypeString,
 					Computed: true,
 				},
-				"memory_per_node": {
+				"size": {
```

Comment:
There's no `size_resource` field 🤔 Also, isn't the value always `memory`? Should we omit this field for all resource types?

Reply:
actually...  It can also be `storage`. See https://github.com/elastic/cloud-sdk-go/blob/master/pkg/models/topology_size.go#L37-L42

Reply:
Ah, got it! Ignore my comments about that then :)